instagram/src/pages/PostModal.jsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import PostAfterSelect from "../components/Post/PostAfterSelect";
+import PostBeforeSelect from "../components/Post/PostBeforeSelect";
+
+function PostModal({ setShowModal }) {
+  const [imgSrc, setImgSrc] = useState("");
+  const [selected, setSelected] = useState(false);
+  const close = () => setShowModal(false);
+  const stopBubble = (e) => e.stopPropagation();
+
+  const Container = ({ children }) => (
+    <Background onClick={close}>
+      <ExitBtn />
+      <Wrap onClick={stopBubble}>
+        <Header>
+          <H1> 새 게시물 만들기</H1>
+        </Header>
+        {children}
+      </Wrap>
+    </Background>
+  );
+
+  return (
+    <>
+      {selected ? (
+        <Container>
+          <PostAfterSelect imgSrc={imgSrc} />
+        </Container>
+      ) : (
+        <Container>
+          <PostBeforeSelect setImgSrc={setImgSrc} setSelected={setSelected} />
+        </Container>
+      )}
+    </>
+  );
+}
+
+export const ExitBtn = styled.button`
+  position: fixed;
+  width: 30px;
+  height: 30px;
+  right: 0;
+  top: 0;
+`;
+
+export const Background = styled.div`
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  background-color: ${({ theme }) => theme.palette.modalBackground};
+  /* cursor: default; */
+`;
+
+export const Wrap = styled.div`
```

Comment:
```suggestion
export const ModalWrapper = styled.div`
```
좀 더 네이밍을 구체적으로하면 좋을 것 같아! 어떤 컴포넌트의 Wrap(Wrapper)인지
이건 프로젝트 전반적으로 신경써야될 부분인듯

Reply:
아할 수정할게 많고만